src/mixins/itext_behavior.mixin.js

```diff
@@ -141,9 +141,15 @@
 
       this.abortCursorAnimation();
       this._currentCursorOpacity = 1;
-      this._cursorTimeout2 = setTimeout(function() {
-        _this._tick();
-      }, delay);
+      clearTimeout(this._cursorTimeout2);
```

Comment:
this should be taken care by `this.abortCursorAnimation`

Reply:
as discusssed @ShaMan123 please remove

Reply:
done 08afa25